src/Fields/Date/index.js

```diff
@@ -25,13 +24,13 @@ const DatePicker = ({
 }) => {
   const [date, setDate] = React.useState(new Date())
   const pickerRef = React.useRef(null)
-  const mapLanguagues = { de, fr, es, en }
-  const datepickerLanguage =
-    language && mapLanguagues[language.toLowerCase()]
-      ? mapLanguagues[language.toLowerCase()]
-      : mapLanguagues.en
+  const mapLanguagues = { de, fr, es }
```

Comment:
why this change?

Reply:
because we dont need the 'en' it is the default so it would have no sense having the else. if it is not in the maplanguages we will show it in english because the locale is 'en' or we will display it in 'en' because it is another random locale and we dont have the language for it.